### ISSUE TITLE
make the pages Responsive ( using Tailwind) 

### ISSUE THREAD

**Issue by codewarnab** [Innovateninjas/Paws-frontend#242]
currently all the pages are mobile friendly make  them desktop frienldy too  , before starting working on any page discuss below get assigned than start working on the page 

if you can make atleast one page responsive your PR will be merged 

**Pages to be Made Responsive** 

- [ ] #262
- [ ] #271
- [x] #261
- [ ] #283
- [ ] campaign form page 
- [ ] #302
- [x] #260
- [x] #282
- [ ] #269
- [ ] #284
- [x] #285
- [ ] #294
- [x] #248
- [ ] #295
- [ ] #280
- [x] #279
- [x] #272
- [ ] #273

Comment below which page you want to make responsive each page will be treated as a separate issue 
Before contributing read `Contribution.md`

**Comment by Ankesh2004** [comment]
I feel that I am suitable for this issue.
Kindly assign it to me under GSSOC'24.

**Comment by m-tabish** [comment]
I would like to do it under GSSOC 24

**Comment by Vaibhavsg17** [comment]
please assign me this issue.

**Comment by NilanchalaPanda** [comment]
Hey @codewarnab, from all the pages that I have explored until now, the CREW details page is not looking good in mobile version. And all the other pages you mentioned are to be optimized for Desktop version. We can work on the issue with you feedback and suggestions. Do consider assigning the issue as a GSSpC'24 Contributor, so that I can start working with Paws.

Also I have worked with Tailwind CSS from the last 1 year.  So I can surely help you with this issue. looking forward to work with Paws 🙌🏻.

**Comment by dikshajaiswal855** [comment]
Hey, @codewarnab, I am eager to work on this issue as I have hands-on experience with tailwind. Do consider assigning the issue to a GSSpC'24 contributor so that I can start working with Paws.

**Comment by rishicds** [comment]
Hi everyone, each page will be a seperate issue so please tell us which page you'd like to work on @Ankesh2004 @Vaibhavsg17 @m-tabish @NilanchalaPanda

**Comment by NilanchalaPanda** [comment]
Hey @rishicds, I would like to work on the CREW page, which is to be optimized for mobile version first. And after completion of that will take any other page.

**Comment by dikshajaiswal855** [comment]
Hey @rishicds , can I work on any page

**Comment by dikshajaiswal855** [comment]
Hey @rishicds, can I work on the NGO profile page? I want to make it appealing for medium or large sized screens as it is already mobile-friendly

**Comment by m-tabish** [comment]
Please Assign me to under GSSOC '24

**Comment by m-tabish** [comment]
> Hi everyone, each page will be a seperate issue so please tell us which page you'd like to work on @Ankesh2004 @Vaibhavsg17 @m-tabish @NilanchalaPanda 

Login page

**Comment by Priyal208** [comment]
Hi I've lots of prev exp working with tailwind and would love to make your pages responsive! Kindly assign me the same! I can work on login page , user reg page and ngo reg page and make them appealing and beautiful whilst maintaing the theme and co-ordination with other components!

**Comment by codewarnab** [comment]
> Hi I've lots of prev exp working with tailwind and would love to make your pages responsive! Kindly assign me the same! I can work on login page , user reg page and ngo reg page and make them appealing and beautiful whilst maintaing the theme and co-ordination with other components!

those pages are already assigned would you like to work on the success page ?

**Comment by Priyal208** [comment]
Yes sure

**Comment by Ankesh2004** [comment]
> Hi everyone, each page will be a seperate issue so please tell us which page you'd like to work on @Ankesh2004 @Vaibhavsg17 @m-tabish @NilanchalaPanda 

User profile page

**Comment by Rehan52004** [comment]
I want to contriubute in one of the issue.

**Comment by Deekshashakyawal** [comment]
could you please assign me an issue, I'm have experience in frontend development

**Comment by Riddhi162** [comment]
Hello Im a GSSOC'24 contributor and a passionate frontend developer i have previously worked with tailwind css, html and javascipt i would love to make the **Animal details** page and **Contact information page** with eye catching and appealing UI can you assign this issues to me.

**Comment by codewarnab** [comment]
this issue is assigned to 

> Hello Im a GSSOC'24 contributor and a passionate frontend developer i have previously worked with tailwind css, html and javascipt i would love to make the **Animal details** page and **Contact information page** with eye catching and appealing UI can you assign this issues to me.

Assigned! Best of luck

**Comment by codewarnab** [comment]
> > Hi everyone, each page will be a seperate issue so please tell us which page you'd like to work on @Ankesh2004 @Vaibhavsg17 @m-tabish @NilanchalaPanda
> 
> User profile page

it has been taken any other page ?

**Comment by codewarnab** [comment]
@Rehan52004 @Deekshashakyawal  name the  pages  you are willing to work on

**Comment by NilanchalaPanda** [comment]
Hey @codewarnab, I have raised PR for Teams Page. Can you assign any other page to work upon. Would love to contribute more. Also can you please review the PR that I have raised. Looking forward to get the following updates

**Comment by codewarnab** [comment]
> Hey @codewarnab, I have raised PR for Teams Page. Can you assign any other page to work upon. Would love to contribute more. Also can you please review the PR that I have raised. Looking forward to get the following updates

yes   I have reviewed, approved your pr it will get merged soon we are making sure all of your contribution is being tracked under GSSOC . 
next on which page are you willing to work ?

**Comment by NilanchalaPanda** [comment]
I would like to Donation page, but can you assist what change you want in this page?
LINK : https://pawss.vercel.app/donation

**Comment by Rehan52004** [comment]
I want to work on Contacts page.

**Comment by codewarnab** [comment]
> I would like to Donation page, but can you assist what change you want in this page? LINK : https://pawss.vercel.app/donation

not sure currently their is so much extra space both side of the page in desktop you have to make sure that look good for the desktop also . Probably explore other app donation pages take some inspiration then start working on it if you want .
let me know if you are willing to do this then  i will assign you this page

**Comment by codewarnab** [comment]
> I want to work on Contacts page.

you mean contact information page it is already assigned

**Comment by Rehan52004** [comment]
Any unassigns page?

**Comment by richab246** [comment]
Hey @codewarnab I want to work on the responsiveness of pages. I am a participant in GSSoC 2024, please assign any page if available to me.

**Comment by dikshajaiswal855** [comment]
Hey @codewarnab , can I work on the NGO profile page? I want to make it appealing for medium or large sized screens as it is already mobile-friendly

**Comment by codewarnab** [comment]
@richab246 @dikshajaiswal855 @Rehan52004 

1. create campaign page 
2. Campaign Form Page 
3. NGO Dashboard 
4. Stats page for Ngo 
5. Campaign List page 
6. Campaign Post Page 
7. Upload image Location page 
feel free to comment  if you face any issue working on this pages 

here are the list of unassigned pages as of now comment below which one you want to work on

**Comment by dikshajaiswal855** [comment]
NGO Dashboard

**Comment by richab246** [comment]
Campaign Form Page or create campaign page

**Comment by dikshajaiswal855** [comment]
Thanks! @codewarnab

**Comment by Deekshashakyawal** [comment]
Campaign List page

**Comment by Aartisushmesh29** [comment]
@codewarnab can I work on the Campaign Post Page? I want to make it responsive appealing for medium or large sized screens as it is already mobile-friendly.Do consider assigning the issue under GSSoC'24 so that I can start working with Paws.

**Comment by Rehan52004** [comment]
I want to work on stats page for ngo

**Comment by dikshajaiswal855** [comment]
@codewarnab please help as I am new to open source contribution


when I have run the command npm start it showed a lot of errors and to fix those errors I got help from chatgbt and then with copilot 😥 but now still, the error is same copilot is saying to delete node modules and package-lock.json and then to remove “webpack” from dependencies/devDependencies.
sir please help what should I do next 
after creating a new branch I have deleted the package-lock.json and tried to delete node modules but it is asking for administrator permission .

![image](https://github.com/Innovateninjas/Paws-frontend/assets/111669785/b9e8fb1c-0024-434b-971f-ad4e2d08892e)

**Comment by dikshajaiswal855** [comment]
@codewarnab I am really eager to contribute and here I need help🤕 please help sir

**Comment by dikshajaiswal855** [comment]
> @codewarnab please help as I am new to open source contribution
> 
> when I have run the command npm start it showed a lot of errors and to fix those errors I got help from chatgbt and then with copilot 😥 but now still, the error is same copilot is saying to delete node modules and package-lock.json and then to remove “webpack” from dependencies/devDependencies. Then it told to Run npm install . sir please help what should I do next after creating a new branch I have deleted the package-lock.json and tried to delete node modules but it is asking for administrator permission .
> 
> ![image](https://private-user-images.githubusercontent.com/111669785/329784482-b9e8fb1c-0024-434b-971f-ad4e2d08892e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU0MzQyNzEsIm5iZiI6MTcxNTQzMzk3MSwicGF0aCI6Ii8xMTE2Njk3ODUvMzI5Nzg0NDgyLWI5ZThmYjFjLTAwMjQtNDM0Yi05NzFmLWFkNGUyZDA4ODkyZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMVQxMzI2MTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01ZGYwZWJlYTI5MGI4N2E5Y2I1MmZmYTg3MzBiYTFhNGZmNDk3ZGM3MjUwYTk4ODE4ZWFhNmI4MWM5ZjBiMDFjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.BqUkPr_F8atE6IskWRdk8fQV4XO86t0z7ObuEKjdmw8)

If any one knows how to solve this issue please help

**Comment by Deekshashakyawal** [comment]
What is the **URL** that i have to enter to reach the **Campaign List Page**  ?? I cannot find the URL , I need to make the page responsive so please reply.

**Comment by GhoshAditi** [comment]
@rishicds assign me upload image page.

**Comment by rishicds** [comment]
done @GhoshAditi

**Comment by NilanchalaPanda** [comment]
> > I would like to Donation page, but can you assist what change you want in this page? LINK : https://pawss.vercel.app/donation
> 
> not sure currently their is so much extra space both side of the page in desktop you have to make sure that look good for the desktop also . Probably explore other app donation pages take some inspiration then start working on it if you want . let me know if you are willing to do this then i will assign you this page



Hey @codewarnab, can you please assign `DONATION PAGE` this issue. I'll work on it. Thanks for the update.

**Comment by anwesha2002** [comment]
hey @codewarnab 
I am a GSSOC'24 contributor . I would like to work on the overall UI , of the signUP page, Login page and the home page. Can you assign this to me, I have previous experience of working with tailwind css, React js.

**Comment by NilanchalaPanda** [comment]
Hey @codewarnab, I have raised a PR for the issue #234 , please review if any changes needed. Looking forward to your reply. Thanks . Also can you please assign me the `DONATION PAGE` to work upon the responsiveness on Desktop version.

**Comment by dikshajaiswal855** [comment]
@codewarnab  @NilanchalaPanda can you help me I am not able to run the command npm start .Please help if you know how to solve this poblem.

![image](https://github.com/Innovateninjas/Paws-frontend/assets/111669785/646ea77a-2b9e-486f-b444-ec9f06dfbfe0)

**Comment by NilanchalaPanda** [comment]
From the directory, you have added some package to the project, because of that this error might have occurred. Mostly it related to you package dependencies. Try uninstalling the package and then again starting the server. Happy Hacking!

**Comment by dikshajaiswal855** [comment]
> From the directory, you have added some package to the project, because of that this error might have occurred. Mostly it related to you package dependencies. Try uninstalling the package and then again starting the server. Happy Hacking!

did you mean to uninstall node-modules and package-lock json and again installing npm

**Comment by NilanchalaPanda** [comment]
Yes you try doing it once, or else uninstall the new package you added and then check. If any more doubts, you can message me on discord username : codey_nil

**Comment by dikshajaiswal855** [comment]
> Yes you try doing it once, or else uninstall the new package you added and then check. If any more doubts, you can message me on discord username : codey_nil

ok thank you

**Comment by codewarnab** [comment]
> I want to work on stats page for ngo

its already taken

**Comment by codewarnab** [comment]
> Can you please assign Campaign List page to me , so that I start working with Paws under GSSoC'24

its in user side path to the file is `/src/pages/user/CampaignPages/CampaignList.jsx`

**Comment by codewarnab** [comment]
> > From the directory, you have added some package to the project, because of that this error might have occurred. Mostly it related to you package dependencies. Try uninstalling the package and then again starting the server. Happy Hacking!
> 
> @NilanchalaPanda thankyou for your reply . But I am still very confused I am trying to solving this problem from yesterday but a new problem arises when I solve 1 error now this error is coming.Can you please help me I am not getting which package to uninstall did you mean to uninstall node-modules and package-lock json and again installing npm

their are some depedency warning you will face during package installation ignore that and dont forget to check 'contribution.md`

**Comment by NilanchalaPanda** [comment]
> Hey @codewarnab, I have raised a PR for the issue #234 , please review if any changes needed. Looking forward to your reply. Thanks . Also can you please assign me the `DONATION PAGE` to work upon the responsiveness on Desktop version.

@codewarnab !?

**Comment by codewarnab** [comment]
> hey @codewarnab I am a GSSOC'24 contributor . I would like to work on the overall UI , of the signUP page, Login page and the home page and I would like to add a navbar as well where people can see their account. Can you assign this to me, I have previous experience of working with tailwind css, React js.

nav bar already desktop optimized  and sign up login page is already assigned any other page you want to work on ?

**Comment by Vaibhavsg17** [comment]
> Hi everyone, each page will be a seperate issue so please tell us which page you'd like to work on @Ankesh2004 @Vaibhavsg17 @m-tabish @NilanchalaPanda

I like to work on NGO Dashboard

**Comment by riya-rjha** [comment]
@rishicds Can I now work on the UI improvisation of buttons of the donation front page. Can you please assign this to me under the Gssoc label?
